devlog/_plan/260817_wave5_execution/090_wave6_closeout.md

```
@@ -41,3 +41,38 @@ resolved: close-on-dev vs close-on-release; #1059 consecutive-green count;
 Cursor non-loopback HTTP; Antigravity undocumented protocol posture;
 needs-info lifetime; upstream-tracker accounting; #1795 recovery shape;
 #1899 disposition; #1836 disposition; #1903 HTTP/1.1 default.
+## Closure policy for THIS run (user decision, 2026-08-18)
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)